src/stackdriver-nozzle/scripts/repro-metrics-rpc-errors.go

```diff
@@ -0,0 +1,70 @@
+package main
```

Comment:
can you add some documentation to this one? eg, what's the error, why we have this code